app/models/document.rb

```diff
@@ -47,6 +48,17 @@ class Document < ApplicationRecord
   validate :ensure_file_attached
 
   scope :common, -> { where(event_id: nil) }
+  scope :general, -> { where(category: :general) }
+  scope :nonprofit_status, -> { where(category: :nonprofit_status) }
+  scope :tax_exception, -> { where(category: :tax_exception) }
+  scope :forms, -> { where(category: :forms) }
```

Comment:
Great use of scopes! Thankfully, Rails takes care of all of these for us: https://api.rubyonrails.org/v5.1.7/classes/ActiveRecord/Enum.html

Reply:
Woah did not know that, removing the unnecessary scopes then!